guide/English/dualboot-en.md

```diff
@@ -55,3 +57,39 @@
   
   - Run the app
   - Press "Quickboot to windows"
+  
+## Finished!
+  
+  
+
+  
+## Dualbooting Android and Windows seamlessly (Option 2)
+  
+> **Note** This may not work on some android kernels or versions but it is easier and faster to boot into android
```

Comment:
another note: sometimes it might break (won't boot to android) and you'll have to reflash it once again